front/app/modules/commercial/analytics/admin/components/EmailDeliveriesCard/useEmailDeliveries/parse.ts

```diff
@@ -52,8 +53,11 @@ const parseRow = (
   };
 };
 
+// TODO: Improve this
+// This isn't the best way to get the date, but the changing attribute name was causing havoc with typings
```

Comment:
Also if there is time, it would be nice if there was a more type safe way of doing this. It works but it doesn't feel the best way to do this.